src/test/resources/fixtures/articles/comment-data.sql

```diff
@@ -0,0 +1,28 @@
+INSERT INTO users
```

Comment:
I think using such fixtures in tests causes a little confusion since we assert on "magic" values defined in other places. When tests are growing over time, these values are shared by multiple of them and doing some changes can easily cause the headache to the developer. IMO better approach is to maintain test data closer to tests (so for example we can track the usage with IDE), tests are more clear, it's harder to do the mess (it's still doable if we have a lot of test data). In addition we don't need to perform so many jumps analysing code. However it requires extra effort to call some "beforeAll"/"before" functions + having test data well-organised.
Two weeks ago I even started removing basic-data.sql and admin.sql but somebody else was refactoring tests so I switched myself to another task and now I'm not sure if this is something we want to do (maybe this is just me and changing it is form over content in case of such small project). It would be good to know other opinions on that (@kciesielski @adamw).

BTW I don't think this is something that blocks this PR - if we decide to change it, then I'd recommend to complete this PR and do the aforementioned changes as separate PR for all tests.

Reply:
Yes, that's a good idea to create the test data from the test methods, and isolate each test case so that it operates on its own data. So 👍

Reply:
I added it as a new issue https://github.com/softwaremill/realworld-tapir-zio/issues/79